addons/docs/src/frameworks/vue/preset.ts

```diff
@@ -1,12 +1,24 @@
 export function webpackFinal(webpackConfig: any = {}, options: any = {}) {
+  let vueDocgenOptions = {};
+
+  options.presetsList.forEach((preset: any) => {
+    if (preset.name.includes('addon-docs') && preset.options.vueDocgenOptions) {
```

Comment:
@shilman 
Should we make sure the preset option works on Storybook monorepo?

The `preset.name` will be `<path to user repo>/node_modules/@storybook/addon-docs/dist/...` on normal installation, but `<path to monorepo>/addons/docs/dist/...` on Storybook monorepo so `preset.name.includes('addon-docs')` won't match. vue-kitchen-sink does not configure `vueDocgenOptions` but will it be confusing in the future?

Reply:
Ugh, as long as this works in user projects I think we'll redo the monorepo soon and hopefully fix this problem.